test/unit/mock/profiles/invalid-include-controls/controls/example.rb

```diff
@@ -0,0 +1,4 @@
+# encoding: utf-8
+# copyright: 2018, The Authors
+
+include_controls 'invalid_name'
```

Comment:
Not a big deal, but you might consider 'no_such_profile' (or similar) rather than 'invalid_name' - directly communicate what is being tested. (There is nothing invalid about 'invalid_name' - you could have a profile named that.)

Reply:
Nope, solid feedback. I couldn't find a good name. Will use `invalid_name`.